cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunNativeImageTestTask.java

```diff
@@ -154,6 +156,12 @@ public RunNativeImageTestTask(PrintStream out, boolean rerunTests, String groupL
             singleExecTests = testList;
         }
         this.listGroups = listGroups;
+
+        if (workers <= 0) {
+            testWorkers = DEFAULT_TEST_WORKERS;
```

Comment:
We should be able to set the default value in the picocli `CommandLine.Option` annotation

Reply:
It is needed only when the worker count is negative or zero. AFAIK, default value we set in picocli is used when we don't set any value. right??